lapis2/src/test/kotlin/org/genspectrum/lapis/model/ModelAggregatedTest.kt

```diff
@@ -0,0 +1,106 @@
+package org.genspectrum.lapis.model
+
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.verify
+import org.genspectrum.lapis.response.AggregatedResponse
+import org.genspectrum.lapis.silo.And
+import org.genspectrum.lapis.silo.PangoLineageEquals
+import org.genspectrum.lapis.silo.SiloAction
+import org.genspectrum.lapis.silo.SiloClient
+import org.genspectrum.lapis.silo.SiloFilterExpression
+import org.genspectrum.lapis.silo.SiloQuery
+import org.genspectrum.lapis.silo.StringEquals
+import org.genspectrum.lapis.silo.True
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+
+class ModelAggregatedTest() {
+    @MockK
+    lateinit var siloClientMock: SiloClient
+
+    @BeforeEach
+    fun setup() {
+        MockKAnnotations.init(this)
+        every { siloClientMock.sendQuery(any<SiloQuery<AggregatedResponse>>()) } returns AggregatedResponse(0)
+    }
+
+    @Test
+    fun `given empty filter parameters then handleRequest should call the SiloClient with MatchAll SiloQuery`() {
+        val aggregatedModel = ModelAggregated(siloClientMock)
```

Comment:
Let's also instantiate this in the `BeforeEach` and make it a field of the test class. That's the usual pattern, it yields cleaner test methods. We could also call it `underTest`, if you like.